src/librustc/traits/trans/mod.rs

```diff
@@ -33,22 +33,22 @@ impl<'a, 'tcx> TyCtxt<'a, 'tcx, 'tcx> {
     /// obligations *could be* resolved if we wanted to.
```

Comment:
Nit: Can we extend this comment to say "/// Assumes that this is run after the entire crate has been successfully type-checked."